presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -639,6 +659,19 @@ existing data in S3::
       external_location = 's3://my-bucket/data/logs/'
     )
 
+Performance
+^^^^^^^^^^^
+It's very easy to query external data, as all you need to do is define the mapping between the data and SQL tables.
```

Comment:
I don't think this belongs in the example section. We have a *Table statistics* section above that talks about `ANALYZE`. We might want to expand the explanation there.

Note that we don't use the phrase "cost based optimizer" in Presto documentation because Presto does not have a true CBO. It has an RBO that makes cost based decisions for join types and orderings.